plugins/woocommerce/client/admin/client/settings-payments/components/payment-settings-layout/payment-settings-layout.scss

```diff
@@ -0,0 +1,22 @@
+.payment-settings-layout {
+	margin: 24px auto 0;
+	max-width: 1032px;
```

Comment:
is that so in design? number seems unusual

Reply:
Have removed this constraint in 370a44a843. Good spot!